src/en/getting-started-21.md

```diff
@@ -0,0 +1,168 @@
+Title: Get started with Juju 2.1
+TODO: add changes in output to model and show-machine
+
+
+# Get started with Juju 2.1
```

Comment:
I totally agree. I was originally going to called it What's new as I was worried about Google picking this up as a general getting started guide. I'll make those changes now.